src/pages/messaging/node-sdk/index.mdx

```diff
@@ -1,47 +1,47 @@
 ---
 weight: 40
 category: "messaging"
-subcategory: "chat-sdk"
-title: "Chat SDK"
+subcategory: "node-sdk"
+title: "LiveChat Node.js SDK"
 subtitle: "App Builder Tool"
-tagline: "Chat SDK was designed to help developers build apps in a quick and easy way."
-desc: "Chat SDK eases the use of our Messaging APIs. Check how to create an app in just a few minutes!"
+tagline: "LiveChat Node.js SDK was designed to help developers build apps in a quick and easy way."
+desc: "LiveChat Node.js SDK eases the use of our Messaging APIs. Check how to create an app in just a few minutes!"
 ---
 
 # Introduction
 
-**Chat SDK** was designed to help developers build apps in a quick and easy way. Under the hood, it makes use of the LiveChat [Messaging APIs](/messaging/), taking care of this part for you.
+**LiveChat Node.js SDK** was designed to help developers build apps in a quick and easy way. Under the hood, it makes use of the LiveChat [Messaging APIs](/messaging/), taking care of this part for you.
 
-By default, Chat SDK comes with a method for sending plain text messages, which is a great starting point for further development.
+By default, this SDK comes with a method for sending plain text messages, which is a great starting point for further development.
 Based on the method template we provide, you can easily build other methods.
 Thanks to that, you have the flexibility to cover the functionalities you need for your app, for example sending rich messages.
 
-If Messaging APIs seem too intimidating or you just want to build something real quick, **Chat SDK** is your way to go.
+If Messaging APIs seem too intimidating or you just want to build something real quick, **LiveChat Node.js SDK** is your way to go.
 
 - <a
     href="https://www.npmjs.com/package/@livechat/chat-sdk/"
     target="_blank"
     rel="noopener noreferrer"
   >
-    Chat SDK on NPM
+    LiveChat Node.js SDK on NPM
```

Comment:
won't we have to rename the GitHub/npm repo, install command, and so on as well?

Reply:
We should but it's too much effort. The long-term plan is for the Developer SDK to absorb Chat SDK